R/p_direction.r

```diff
@@ -7,10 +7,11 @@ p_direction.lm <- function(x, dof = NULL, ...) {
   if (is.null(dof)) {
     dof <- degrees_of_freedom(x)
   }
-  out <- .data_frame(
-    Parameter = stats$Parameter,
-    pd = stats::pt(abs(stats$Statistic), df = dof, lower.tail = TRUE)
-  )
+  pd <- stats::pt(abs(stats$Statistic), df = dof, lower.tail = TRUE)
+  if (any(pd < 0.5)) {
```

Comment:
This is not needed - since you're using `abs`, the smallest this can be is 0.5.

Reply:
ah, ok!